framework/messages/hi/yii.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+/**
+ * Message translations.
+ *
+ * This file is automatically generated by 'yii message/extract' command.
+ * It contains the localizable messages extracted from source code.
+ * You may modify this file by translating the extracted messages.
+ *
+ * Each array element represents the translation (value) of a message (key).
+ * If the value is empty, the message is considered as not translated.
+ * Messages that no longer need translation will have their translations
+ * enclosed between a pair of '@@' marks.
+ *
+ * Message string can be used with plural forms format. Check i18n section
+ * of the guide for details.
+ *
+ * NOTE: this file must be saved in UTF-8 encoding.
+ */
+return [
+    ' and ' => ' और ',
+    '(not set)' => '(स्थापित नहीं)',
+    'An internal server error occurred.' => 'सर्वर की एक आंतरिक खामी हुई।',
+    'Are you sure you want to delete this item?' => 'क्या आप सुनिश्चित रूप से इस चीज को नष्ट करना चाहते हैं?',
+    'Delete' => 'नष्ट',
+    'Error' => 'खामी',
+    'File upload failed.' => 'फ़ाइल अपलोड असफल हुआ।',
+    'Home' => 'घर',
+    'Invalid data received for parameter "{param}".' => 'पैरामीटर "{param}" के लिए अमान्य माहिती मिली।',
+    'Login Required' => 'लॉगिन आवश्यक',
+    'Missing required arguments: {params}' => 'आवश्यक तर्क: {params} अनुपस्थित है',
+    'Missing required parameters: {params}' => 'आवश्यक पैरामीटर: {params} अनुपस्थित है',
+    'No' => 'नहीं',
+    'No results found.' => 'कोई परिणाम नहीं मिला।',
+    'Only files with these MIME types are allowed: {mimeTypes}.' => 'इन MIME प्रकारों वाली केवल फ़ाइलों की अनुमति है: {mimeTypes}।',
+    'Only files with these extensions are allowed: {extensions}.' => 'इन एक्सटेंशन वाली केवल फाइलों की अनुमति है: {extensions}।',
+    'Page not found.' => 'पृष्ठ नहीं मिला।',
+    'Please fix the following errors:' => 'कृपया निम्नलिखित खामीयां सुधारें:',
+    'Please upload a file.' => 'कृपया एक फ़ाइल अपलोड करें।',
+    'Powered by {yii}' => '{Yii} द्वारा संचालित',
```

Comment:
@samdark 

After your comment (https://github.com/yiisoft/yii2/pull/18023#discussion_r419454651), I am checking all translation again. And found out that in this(Line no:45) translation msg, EN lang has `yii` all in small letters and in HI translation `Y` is capital. Does not there should be Y small on the right side in HI msg? I think it should be small but I am bit confused and need your opinion.

Reply:
Yes. These parameters should stay exactly as is.

Reply:
Can you please help me to better understnad why it should stay exactly as it is?

Reply:
Because it is a parameter. It is replaced to actual value. If it's different, it won't be replaced but will stay as `{Yii}` in the final message.